docs/writing-stories/introduction.md

```diff
@@ -240,7 +244,8 @@ You can also reuse stories from the child `ListItem` in your `List` component. T
     'react/list-story-reuse-data.js.mdx',
     'react/list-story-reuse-data.ts.mdx',
     'angular/list-story-reuse-data.ts.mdx',
-    'vue/list-story-reuse-data.js.mdx'
+    'vue/list-story-reuse-data.vue2.js.mdx',
+     'vue/list-story-reuse-data.vue3.js.mdx',
```

Comment:
```suggestion
    'vue/list-story-reuse-data.vue2.js.mdx',
    'vue/list-story-reuse-data.vue3.js.mdx',
```

Reply:
@phated fixed with the latest commit. Thanks for the clarification.